packages/components/src/index.js

```diff
@@ -184,3 +184,4 @@ export {
 } from './higher-order/with-focus-return';
 export { default as withNotices } from './higher-order/with-notices';
 export { default as withSpokenMessages } from './higher-order/with-spoken-messages';
+export { confirm } from './confirm'; // @todo make it experimental
```

Comment:
Just noting and agreeing with the TODO here — this component should be marked as Experimental

Reply:
Yeah, I'll do that before promoting this to a PR. Thanks for the reminder!